lib/actions.js

```diff
@@ -59,6 +65,9 @@ module.exports = {
 
   isEmailSendingAction: function isEmailSendingAction(action) {
     return EMAIL_SENDING_ACTION[action]
-  }
+  },
 
+  isSMSSendingAction: function isSMSSendingAction(action) {
```

Comment:
If you switched to ES6 style here, we could end the senseless tyranny of double-named functions that's been blighting this module.